framework/validators/UniqueValidator.php

```diff
@@ -69,6 +78,9 @@ public function init()
         if ($this->message === null) {
             $this->message = Yii::t('yii', '{attribute} "{value}" has already been taken.');
         }
+	    if ($this->comboNotUnique === null) {
+		    $this->comboNotUnique = Yii::t('yii', 'Combination [{valueCombo}] of [{attributeCombo}] has already been taken.');
```

Comment:
"messages" folder of the framework does not have a subfolder for English language. I think it is sufficient to specify English strings directly in the code using `Yii::t()` and then "messages/XX/yii.php" files for each language XX are generated automatically using `yii message` command line command.